src/libraries/Microsoft.PowerFx.LanguageServerProtocol/Protocol/CodeActionResultContext.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+namespace Microsoft.PowerFx.LanguageServerProtocol.Protocol
+{
+    /// <summary>
+    /// CodeAction context object model.
+    /// </summary>
+    public class CodeActionResultContext
```

Comment:
>CodeActionResultContext [](http://example.com/codeflow?start=17&length=23)

This looks promising. 

Can you add a test that shows how this might be used? 
Ie, can we set ProviderName/ActionIdentifier to any values when we create the CodeFix, and then we just receive them back when the fix is applied?